prime-angular-frontend/src/app/modules/gis-enrolment/shared/services/gis-enrolment.service.ts

```diff
@@ -37,7 +37,7 @@ export class GisEnrolmentService implements IEnrolmentService<GisEnrolment> {
   }
 
   public get enrolment$(): Observable<GisEnrolment> {
-    // Allow subscriptions, but make immutable
+    // Allow subscriptions, but make the subject immutable
```

Comment:
subject or object?

Reply:
No, it's a BehaviourSubject being made immutable